dex-template/__DexName__.ts.template

```diff
@@ -0,0 +1,224 @@
+import { Interface } from '@ethersproject/abi';
+import { DeepReadonly } from 'ts-essentials';
+import {
+  Token,
+  Address,
+  ExchangePrices,
+  Log,
+  AdapterExchangeParam,
+  SimpleExchangeParam,
+  PoolLiquidity,
+  Logger,
+} from '../../types';
+import {
+  SwapSide,
+  Network,
+} from '../../constants';
+import { StatefulEventSubscriber } from '../../stateful-event-subscriber';
+import { wrapETH, getDexKeysWithNetwork } from '../../utils';
+import { IDex } from '../../dex/idex';
+import { IDexHelper } from '../../dex-helper/idex-helper';
+import {
+  __DexName__Data,
+  PoolState,
+} from './types';
+import { SimpleExchange } from '../simple-exchange';
+import { __DexName__Config } from './config';
+
+const Adapters: { [chainId: number]: { name: string; index: number }[] } = {
+  // TODO: add adapters for each chain 
+};
```

Comment:
This should go in the config.ts according to other changes in this PR.  The config.ts should be generated too.

Reply:
fixed